src/features/inventory/InventoryPreview.tsx

```diff
@@ -0,0 +1,90 @@
+import { useActor } from "@xstate/react";
+import classNames from "classnames";
+import React, { useContext, useState } from "react";
+
+import { Box } from "components/ui/Box";
+import { OuterPanel } from "components/ui/Panel";
+import { Context } from "features/game/GameProvider";
+import { InventoryItemName } from "features/game/types/game";
+import { ITEM_DETAILS } from "features/game/types/images";
+import { Craftable } from "features/game/types/craftables";
+import { SeedName, SEEDS } from '../game/types/crops';
+import { NFTs, TOOLS, FOODS } from "features/game/types/craftables";
+
+interface Props {
+    items: Object
+  }
+
+export const InventoryPreview: React.FC<Props> = ({ items }) => {
+    
+    // let items2 = Object.assign(TOOLS)
+    // localStorage.setItem("inventory.tab", items);
+
+    const { gameService, selectedItem, shortcutItem } = useContext(Context);
+    const [game] = useActor(gameService);
+    const inventory = game.context.state.inventory;
+
+    const current_items = Object.keys(inventory) as InventoryItemName[];
```

Comment:
Mixture between camelcase and snake case. Consider using camelCase for consistency.

E.g. `currentItems`